components/src/atoms/password-strength/PasswordStrength.spec.js

```diff
@@ -0,0 +1,54 @@
+import { mount } from '@vue/test-utils'
+import APasswordStrength from './PasswordStrength.vue'
+
+describe('PasswordStrength', () => {
+  it('has default structure', () => {
+    const wrapper = mount(APasswordStrength)
+    const $passwordIndicator = wrapper.find('.a-password-strength__indicator')
+
+    expect(wrapper.is('div')).toBe(true)
+    expect(wrapper.classes()).toContain('a-password-strength')
+    expect(wrapper.classes().length).toBe(1)
+    expect($passwordIndicator.attributes('aria-live')).toBeDefined()
```

Comment:
Is it needed if `toEqual` will return false if value will be undefined - empty attribute?